### PR TITLE
fix(client): preview account tooltips

### PR DESCRIPTION
**Pull request by abrl91**
Close: https://github.com/amplication/private-issues/issues/123

## PR Checklist

- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/amplication-client/src/Components/FeatureIndicatorContainer.tsx [hunk 1]
```diff
@@ -101,7 +101,15 @@ export const FeatureIndicatorContainer: FC<Props> = ({
     }
 
     return featureText;
-  }, [disabled]);
+  }, [disabled, featureText, limitationText]);
```
Comment (morhag90) [inline packages/amplication-client/src/Components/FeatureIndicatorContainer.tsx:104]: Why add the featureText and the limitationText if it comes from the props?

File: packages/amplication-client/src/Components/LicenseIndicatorContainer.tsx [hunk 2]
```diff
@@ -106,6 +107,15 @@ export const LicenseIndicatorContainer: FC<Props> = ({
     licensedResourceType,
   ]);
 
+  const linkText = useMemo(() => {
+    if (isPreviewPlan(subscriptionPlan)) {
+      console.log("isPreviewPlan");
```
Comment (morhag90) [inline packages/amplication-client/src/Components/LicenseIndicatorContainer.tsx:112]: remove log

File: packages/amplication-client/src/Components/FeatureIndicatorContainer.tsx (hunk 1 above)
Comment (abrl91) [reply packages/amplication-client/src/Components/FeatureIndicatorContainer.tsx:104]: the eslint rule told me to add it :)

File: packages/amplication-client/src/Components/LicenseIndicatorContainer.tsx (hunk 2 above)
Comment (abrl91) [inline packages/amplication-client/src/Components/LicenseIndicatorContainer.tsx:112]:
```suggestion
```

**Review by morhag90 — APPROVED** [review]

**Review by morhag90 — APPROVED** [review]

**Pull request merged**